library/portable-simd/crates/core_simd/src/lib.rs

```diff
@@ -7,7 +7,8 @@
     repr_simd,
     simd_ffi,
     staged_api,
-    stdsimd
+    stdsimd,
+    include_macros
```

Comment:
Probably not needed but I just added this to the feature-set of all the crates that used one of these macros internally.